blog/app/Http/Controllers/Tag/TagController.php

```diff
@@ -85,12 +84,14 @@ public function edit($id)
      */
     public function update(TagRequest $request, $id)
     {
-        $data = [
-            'tag_name' => $request->input('tag_name'),
-        ];
-        Tag::where('id', $id)->update($data);
-
-        return redirect()->route('tags.index');
+        $tag = Tag::find($id);
```

Comment:
Đoạn này sửa thành  thì ko cần check `$tag` null nữa
```suggestion
        $tag = Tag::findOrFail($id);
```

Reply:
E muốn check null để quay lại page trước. Như vậy được không a nhỉ